src/intl/en/page-layer-2.json

```
@@ -40,19 +40,19 @@
-  "layer-2-how-does-layer-2-work-2": "A layer 2 blockchain regularly submits bundles of transactions to Ethereum. Once that happens, reverting layer-2 transactions requires reverting Ethereum. This is what is meant by \"inherited security\". Scaling via layer 2 requires no changes to the layer 1 protocol (Ethereum). This lets layer 1 handle security, data availability, and decentralization, while layer 2s handles scaling. Layer 2s take the transactional burden away from the layer 1 and post finalized proofs back to the layer 1. By removing this transaction load from layer 1, the base layer becomes less congested, and everything becomes more scalable.",
+  "layer-2-how-does-layer-2-work-1": "As we mentioned above, layer 2 is a collective term for Ethereum scaling solutions that handle transactions off Ethereum layer 1 while still taking advantage of the robust decentralized security of Ethereum layer 1. <strong>A layer 2 is a separate blockchain that extends Ethereum</strong>. How does that work?",
+  "layer-2-how-does-layer-2-work-2": "There are several different types of layer 2, each having their own trade-offs and trust assumptions. Layer 2s take the transactional burden away from the layer 1 allowing it to become less congested, and everything becomes more scalable.",
   "layer-2-rollups-title": "Rollups",
-  "layer-2-rollups-1": "Rollups are currently the preferred layer 2 solution for scaling Ethereum. By using rollups, users can  <a href=\"https://l2fees.info/\">reduce gas fees by up to 100x</a> compared to layer 1.",
-  "layer-2-rollups-2": "Rollups bundle (or ’roll up’) hundreds of transactions into a single transaction on layer 1. This distributes the L1 transaction fees across everyone in the rollup, making it cheaper for each user. Rollup transactions get executed outside of layer 1 but the transaction data gets posted to layer 1. By posting transaction data onto layer 1, rollups inherit the security of Ethereum. There are two different approaches to rollups: optimistic and zero-knowledge - they differ primarily on how this transaction data is posted to L1.",
+  "layer-2-rollups-1": "Rollups bundle (or ’roll up’) hundreds of transactions into a single transaction on layer 1. This distributes the L1 transaction fees across everyone in the rollup, making it cheaper for each user.",
+  "layer-2-rollups-2": "Rollup transactions get executed off layer 1 but the transaction data gets posted to layer 1. By posting transaction data onto layer 1, rollups inherit the security of Ethereum. This is because once the data is posted to layer 1, reverting a rollup transaction requires reverting Ethereum. There are two different approaches to rollups: optimistic and zero-knowledge - they differ primarily on how this transaction data is posted to L1.",
```

```suggestion
  "layer-2-rollups-2": "Rollup transactions get executed off layer 1 but the transaction data is submitted to layer 1. By submitting transaction data onto layer 1, rollups inherit the security of Ethereum. This is because once the data is uploaded to layer 1, reverting a rollup transaction requires reverting Ethereum. There are two different approaches to rollups: optimistic and zero-knowledge - they differ primarily on how this transaction data is submitted to L1.",
```

"to post" is dev jargon, language like "submit" contains same message, but is much more user friendly.